frb_codegen/src/main.rs

```diff
@@ -49,6 +49,11 @@ mod tests {
         static ref LOGGER: () = init_logger(".", true).unwrap();
     }
 
+    // #[cfg(windows)]
+    // const DART: &str = "dart.bat";
+    // #[cfg(not(windows))]
+    const DART: &str = "dart";
```

Comment:
nit: hmm so it is the same as the old code?

Reply:
On my machine, I have to use `dart.bat` to run tests successfully. 

But on CI windows machine, `dart.bat` causes `not found` error.

So I have to comment out `dart.bat` to pass the CI.

Maybe flutter version matters.

Reply:
Hmm which flutter are you using?

Reply:
`flutter doctor` :

![image](https://user-images.githubusercontent.com/59032810/236679640-42cba882-2db5-47f4-a8d0-118748d2cff0.png)